frontend/src/features/admin-form/common/components/PreviewFormBanner/PreviewFormBanner.tsx

```diff
@@ -127,12 +128,21 @@ export const PreviewFormBanner = ({
             </Button>
           )}
         </Flex>
-        <DuplicateFormModal
-          formId={formId}
-          isTemplate
-          isOpen={isModalOpen}
-          onClose={onModalClose}
-        />
+        {isTemplate ? (
+          <DuplicateFormTemplateModal
```

Comment:
nit on naming: seems quite similar to `DuplicateFormModal`. maybe we can mirror the endpoint terminology and call it `UseTemplateModal`?